packages/shared/toShallowRef/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { ComputedRef, Ref, ShallowRef } from 'vue-demi'
+import {
+  isRef,
+  shallowRef,
+} from 'vue-demi'
+import type { MaybeRefOrGetter } from '../utils'
+import { toRef } from '../'
```

Comment:
```suggestion
import { toRef } from '..'
```

Reply:
This suggestion fails the linter requirements, but I did expand it to `import { toRef } from '../toRef'` to be clearer. I'm not sure if that would be related to the reason tests were failing.

```
ESLint: '..' import is restricted from being used.(no-restricted-imports)
```